kindlecomicconverter/comic2ebook.py

```diff
@@ -1056,6 +1056,8 @@ def makeParser():
                                      "2: Consider every subdirectory as separate volume [Default=0]")
     output_options.add_argument("--spreadshift", action="store_true", dest="spreadshift", default=False,
                                 help="Shift first page to opposite side in landscape for spread alignment")
+    output_options.add_argument("--filefusion", action="store_true", dest="filefusion", default=False,
+                                help="Combines all selected files into a single file. (Helpful for combining chapters into volumes)")
```

Comment:
File Fusion should be GUI exclusive feature based on how it works.

The GUI and CLI both call `makeBook()` on each file. There's no way to pass the CLI a list of files.